packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
We should do a version bump on this; we only use `NEXT` when we don't want to publish changes (e.g., test-only changes)